### ISSUE TITLE
Find and replace does not work for whole directories

### ISSUE THREAD

**Issue by MeerKatDev** [atom/atom#22028]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

The find and replace doesn't work anymore

### Steps to Reproduce

Open search tab, try to search something in a subdirectory

**Expected behavior:**

To find text

Finds nothing at all, and searches no path too

always

### Versions

atom version 1.54.0
on macOS Big Sur v11.1

### Additional Information

Not much, just that. No packages interacting with it. Does the same in safe mode

<img width="1440" alt="Screenshot 2021-03-13 at 11 14 54" src="https://user-images.githubusercontent.com/1340996/111027088-549d9280-83ee-11eb-804f-7960491c7773.png">

**Comment by ThatXliner** [comment]
I'm pretty sure it asks for a directory, not a file glob

**Comment by ThatXliner** [comment]
Yeah so I'm pretty sure Atom thinks you're searching for the directory named `/some/random/thing/so/long/*.php` instead of a glob... probably a bug
